frontend/packages/kubevirt-plugin/integration-tests/tests/cnv-uninstallation.scenario.ts

```diff
@@ -0,0 +1,26 @@
+import * as sidenavView from '@console/internal-integration-tests/views/sidenav.view';
+import { isLoaded } from '@console/internal-integration-tests/views/crud.view';
+import { $, browser, by, element } from 'protractor';
+import { click } from '@console/shared/src/test-utils/utils';
+
+const namespaceButton = $('.co-namespace-selector button');
+const openshiftNamespaceButton = $('#openshift-cnv-link');
+ 
+describe('Uninstall Kubevirt', () => {
+ 
+  beforeAll(async () => {
+    await sidenavView.clickNavLink(['Operators', 'Installed Operators']);
+    await isLoaded();
+    await click(namespaceButton);
+    await click(openshiftNamespaceButton);
```

Comment:
It would be better to call `await isLoaded();` here instead of calling it in the test case

Reply:
like to move "await isLoaded();" to line 16?